docs/apache-airflow/start.rst

```diff
@@ -149,6 +149,9 @@ the all-in-one ``standalone`` command, you can instead run:
 
     airflow triggerer
 
+.. note::
+    ``airflow users`` command is only available when `FAB auth manager <https://airflow.apache.org/docs/apache-airflow-providers-fab/stable/auth-manager/index.html>`_ is enabled.
```

Comment:
I think it is better to link the file doc directly. So that if it gets renamed or removed, you'll be aware (doc generation will fail). I dont think it is possible to do it in `contributing-docs` because they are 2 different doc repos

```suggestion
    ``airflow users`` command is only available when :doc:`apache-airflow-providers-fab:auth-manager/index` is enabled.
```

Reply:
Thanks, I fixed it.